inst/qml/jaspPlotBuilder.qml

```diff
@@ -0,0 +1,2932 @@
+//
+// Copyright (C) 2013-2024 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+import QtQuick
+import QtQuick.Layouts
+import JASP
+import JASP.Controls
+
+
+Form {
+    columns: 1
+
+    infoBottom:
```

Comment:
Two things.

1. This looks generated. If it is, can you add the R code to generate this as a comment above this for future updates?
2. You can do string continuation with `\` (see https://forum.qt.io/topic/113667/multiline-strings/2). Not sure how to handle `\n` though.